tests/fe/bdm_16.cc

```diff
@@ -0,0 +1,63 @@
+// ---------------------------------------------------------------------
+//
+// Copyright (C) 2003 - 2017 by the deal.II authors
```

Comment:
Ah, ok, I was just trying to be consistent with the other files' copyright formats, I will change it 😄